src/benchmarks/micro/libraries/Microsoft.Extensions.Configuration/GetChildKeysTests.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Memory;
+using BenchmarkDotNet.Attributes;
```

Comment:
The CI is failing with "error CS0103: The name 'Categories' does not exist in the current context"

```suggestion
using BenchmarkDotNet.Attributes;
using MicroBenchmarks;
```